src/powersystem.jl

```diff
@@ -240,11 +240,13 @@ Returns vectors indexing buses by type: ref, pv, pq.
 """
 function bustypeindex(bus, gen)
     # retrieve indeces
-    BUS_B, BUS_AREA, BUS_VM, BUS_VA, BUS_NVHI, BUS_NVLO, BUS_EVHI,
-    BUS_EVLO, BUS_TYPE = Parse.idx_bus()
+    BUS_I, BUS_TYPE, PD, QD, GS, BS, BUS_AREA, VM, VA, BASE_KV, ZONE, VMAX, VMIN,
+    LAM_P, LAM_Q, MU_VMAX, MU_VMIN = IdxSet.idx_bus()
+    
+    GEN_BUS, PG, QG, QMAX, VG, MBASE, GEN_STATUS, PMAX, PMIN, PC1, PC2, QC1MIN,
+    QC2MIN, QC2MAX, RAMP_AGC, RAMP_10, RAMP_30, RAMP_Q, APF, MU_PMAG, MU_PMIN, MU_QMAX,
+    MU_QMIN = IdxSet.idx_gen()
```

Comment:
I would suggest using a named tuple there, to simplify the management of the elements returned by `idx_gen` :-)

Reply:
Yes, 

I think we should use dictionaries instead of what we are doing right now. So:

data["gen"][1]["GEN_BUS"]

which is something similar to what PowerModels.jl has. This, as long as these dicts are not accessed in performance-critical parts of the code.

Again, lets discuss it after we get some sensible results for RGM.